sycl/doc/extensions/proposed/sycl_ext_intel_cluster_properties.asciidoc

```diff
@@ -0,0 +1,101 @@
+= sycl_ext_intel_cluster_properties
+
+== Introduction
+
+On FPGA targets, regions of the circuit called clusters may be statically scheduled.
+This extension introduces two kernel properties that specify how statically-scheduled clusters should be implemented for an FPGA target.
+
+== Notice
+
+Copyright (c) 2022 Intel Corporation.  All rights reserved.
+
+== Status
+
+Working Draft
+
+This is a proposed extension specification, intended to gather community
+feedback. Interfaces defined in this specification may not be implemented yet
+or may be in a preliminary state. The specification itself may also change in
+incompatible ways before it is finalized. Shipping software products should not
+rely on APIs defined in this specification.
+
+== Version
+
+Last Modified Date: 2022-12-19 +
+Revision: 1
+
+== Contributors
+
+Jessica Davies, Intel +
+
+== Dependencies
+
+This extension is written against the SYCL 2020 specification, Revision 6 and
+the following extension:
+
+- link:sycl_ext_oneapi_kernel_properties.asciidoc[sycl_ext_oneapi_kernel_properties]
```

Comment:
The link is broken, probably because the extension was moved from "proposed" to "experimental".